dev/com.ibm.ws.ras.instrument/src/com/ibm/ws/ras/instrument/internal/main/DynamicTraceInstrumentation.java

```diff
@@ -150,22 +167,23 @@ class Transformer extends StaticTraceInstrumentation implements ClassFileTransfo
         }
     }
 
-    /**
-     * Instrument the classes.
-     */
     public byte[] transform(ClassLoader loader,
                             String className,
                             Class<?> classBeingRedefined,
                             ProtectionDomain protectionDomain,
-                            byte[] classfileBuffer) throws IllegalClassFormatException {
+                            byte[] classBytes) throws IllegalClassFormatException {
 
+        String methodName = "transform";
+        
         // Don't modify our own package
         if (className.startsWith(Transformer.class.getPackage().getName().replaceAll("\\.", "/"))) {
```

Comment:
I thought you were looking at changing this to contains instead of startsWith?  If it is a full URI path, it will include additional content than just the relative path to the file.  It may be better to strip off the .classpath and do an endsWith?

Also if it is a className instead of a path, why are we convert the package name to have slashes instead of periods?

Reply:
This occurrence is really a class name.  Entry is from 'com.ibm.ws.classloading.internal.AppClassLoader.doTransformation(String, byte[], ClassFileTransformer)'.
The prior update to change this to 'path' was incorrect.  This method should not have been updated.

Reply:
Actually will AppClassLoader.doTransformation() even use this Transformer?  I didn't think we did trace transformation for application classes.  I thought we only did it for run time classes.  If so this isn't a problem for this PR, but is a problem still for the transformers that are called by AppClassLoader and can be done under a separate PR.